samples/wifi/shutdown/Kconfig

```
@@ -13,3 +13,34 @@ config WIFI_MAC_ADDRESS
 		WiFi MAC address to be used by the sample. If not set, the sample will
 		use the default MAC address (locally administrative). This is only used
 		when the nRF7002 OTP is not programmed with a MAC address.
+
+choice SHUTDOWN_OPERATION_MODE
+	prompt "Sample operation mode"
+	default OPERATION_MODE_CONTINUOUS
+	help
+		Select the operation mode for the sample.
+
+config OPERATION_MODE_CONTINUOUS
+	bool "Continuous mode"
+	help
+		In this mode, the sample will continue to run alternating between
```

```suggestion
	In this mode, the sample will continue to run alternating between
```
Indent is just two spaces  for help.